src/containers/Simularium/index.tsx

```diff
@@ -103,8 +108,12 @@ class App extends React.Component<AppProps, AppState> {
             }
         } else if (userTrajectoryUrl) {
             const verifiedUrl = urlCheck(userTrajectoryUrl);
+            let googleDriveId;
+            if (isGoogleDriveUrl(verifiedUrl)) {
+                googleDriveId = getGoogleDriveFileId(verifiedUrl, parsed.id);
+            }
```

Comment:
I would consider pushing all of lines 110-114 back into userUrlHandling, to hide some of the details from this module.  (AKA "Why does this module need to know anything about googleDriveId?")